tests/datasource_test.go

```diff
@@ -65,20 +67,40 @@ var _ = Describe("DataSource", func() {
 	}
 
 	It("[test_id:8041]status conditions should be updated on pvc create/update/delete", func() {
-		By("creating datasource")
+		By("Create DataSource with no source PVC")
 		ds := newDataSource(ds1Name)
 		ds, err := f.CdiClient.CdiV1beta1().DataSources(ds.Namespace).Create(context.TODO(), ds, metav1.CreateOptions{})
 		Expect(err).ToNot(HaveOccurred())
 		ds = waitForReadyCondition(ds, corev1.ConditionFalse, "NoPvc")
 
+		By("Update DataSource source PVC to nonexisting one")
 		ds.Spec.Source.PVC = &cdiv1.DataVolumeSourcePVC{Namespace: f.Namespace.Name, Name: pvc1Name}
 		ds, err = f.CdiClient.CdiV1beta1().DataSources(ds.Namespace).Update(context.TODO(), ds, metav1.UpdateOptions{})
 		Expect(err).ToNot(HaveOccurred())
 		ds = waitForReadyCondition(ds, corev1.ConditionFalse, "NotFound")
 
+		By("Create clone DV with SourceRef pointing the DataSource")
+		dv := utils.NewDataVolumeWithSourceRef("clone-dv", "1Gi", ds.Namespace, ds.Name)
+		dv.Annotations[controller.AnnImmediateBinding] = "true"
```

Comment:
We are not using `ForceBindPvcIfDvIsWaitForFirstConsumer` anymore to ensure the right thing happens in WFFC case? I don't mind just adding the annotation as it is simpler, just asking.

Reply:
I personally prefer the simpler DV annotation instead of WaitForPVC & ForceBind, and it'salso used in kubevirt DIC/DVTemplates etc.

Reply:
Okay